client/trino-cli/src/main/java/io/trino/cli/ClientOptions.java

```diff
@@ -190,8 +194,8 @@ public class ClientOptions
     public Optional<String> clientInfo;
 
     @PropertyMapping(CLIENT_TAGS)
-    @Option(names = "--client-tags", paramLabel = "<tags>", description = "Client tags")
-    public Optional<String> clientTags;
```

Comment:
This is an actual set rather than a single string. This is now converted using ClientTagsConverter